packages/ckeditor5-engine/docs/framework/guides/deep-dive/schema.md

```diff
@@ -514,6 +525,91 @@ Taking this even further, if anyone registers a `<section>` element (with the `a
 	You can read more about the format of the item definition in {@link module:engine/model/schema~SchemaItemDefinition}.
 </info-box>
 
+### Generic items allowed structure
+
+Generic items could be visualized by the following structure (based on `allowIn`, `allowWhere`, etc.):
+
+```xml
+<$root>
+	<$block>                <!-- example: <paragraph>, <heading1> -->
+		<$text/>
+		<$inlineObject/>    <!-- example: <imageInline> -->
+	</$block>
+	<$blockObject/>         <!-- example: <imageBlock>, <table> -->
+	<$container>            <!-- example: <blockQuote> -->
+		<$container/> 
+		<$block/>
+		<$blockObject/>
+	</$container>
+</$root>
+```
+
+The above abstract structure with some generic items implemented could look like:
```

Comment:
```suggestion
The above structure is an abstract one, as it bases on generic items. It can be translated to the following structure when dealing with real model elements.
```